docs/install/index.md

```diff
@@ -6,6 +6,11 @@ nav_order: 1
 redirect_from: /docs/how-to/install
 ---
 
+# Install Dependencies before installing ONNX Runtime
```

Comment:
I think it would be better to add one line in the code block of each installation variant. Then users can copy paste the code block in one click